js/models/stock.js

```diff
@@ -1,12 +1,38 @@
-// define your Stock model here
 function Stock(symbol) {
   this.symbol = symbol;
   this.numShares = 0;
 
+
   // your code to search the API and return a Stock object here
 
 }
 
+
 Stock.prototype = {
-  totalValue: function() { return this.price * this.numShares; }
-}
+  totalValue: function(){
+  	return this.price * this.numShares;
+  },
+  refresh: function(){
```

Comment:
I'm curious why you decided to call this function `refresh`, what is the nature of the function, what is it doing for you?